lib/main.coffee

```diff
@@ -34,7 +37,7 @@ module.exports =
             line = parseInt(match[1]) or 0
             col = parseInt(match[2]) or 0
             toReturn.push({
-              type: "Error"
+              type: if atom.config.get('linter-pep8.convertAllErrorsToWarnings') then 'Warning' else 'Error'
```

Comment:
How about you store this config value somewhere in lint method, instead of checking for each message